gravitybridge/assetlist.json

```diff
@@ -375,6 +375,48 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/occamfi.png"
         }
       ]
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "denom_units": [
+        {
+          "denom": "gravity0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e",
+          "exponent": 0
+        },
+        {
+          "denom": "page",
+          "exponent": 8
+        }
+      ],
+      "base": "gravity0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e",
+      "name": "Page",
+      "display": "Page",
```

Comment:
Please correct the 'display' value to "page" (lowercase 'p', as that's what you have defined as a denom_unit)